src/main/java/MaxMinAverage.java

```diff
@@ -0,0 +1,60 @@
+/*
+1.1. Максимальное, минимальное и среднее значение
+Задача:
+Заполните массив случайным числами и выведете максимальное, минимальное и среднее значение.
+Для генерации случайного числа используйте метод Math.random().
+*/
+
+
+import java.util.ArrayList;
+
+public class MaxMinAverage {
+
+    public static void main (String[] args) {
+        System.out.println("--------1.1 - Max, Min, Average values--------");
+        ArrayList<Integer> createdArray =  generateRandomArray(20);    //create array and fill it by 10 random
+        // int numbers from 0 to 10
+        System.out.println("Max value of array " + createdArray + " is: " + MaxMinAverage.getMaxArrayValue(createdArray));
+        System.out.println("Min value of array " + createdArray + " is: " + MaxMinAverage.getMinArrayValue(createdArray));
+        System.out.println("Average value of array " + createdArray + " is: " + MaxMinAverage.getAverageArrayValue(createdArray));
+    }
+
+    //method for array generating and filling it by random int numbers from 0 to 10
+    static ArrayList<Integer> generateRandomArray (int arrayLength) {
+        ArrayList<Integer> newArray = new ArrayList<>();
+           for (int i = 0; i < arrayLength; i++) {
+                newArray.add(i, (int) (Math.random() * 10));
+            }
+            return newArray;
+    }
+
+    //method for getting max value of input array
+    private static int getMaxArrayValue (ArrayList<Integer> inputArray) {
+         int maxArrayValue = inputArray.get(0);
+         for (Integer integer : inputArray) {
+             if (integer > maxArrayValue)
+             maxArrayValue = integer;
```

Comment:
1) indentation?
2) it is better to avoid omitting brackets even for a single statement, as it may potentially grow in future and developer might miss the scope